providers/src/airflow/providers/standard/operators/weekday.py

```diff
@@ -39,7 +39,8 @@ class BranchDayOfWeekOperator(BaseBranchOperator):
     .. code-block:: python
 
         from airflow.operators.empty import EmptyOperator
-
+        from airflow.operators.weekday import BranchDayOfWeekOperator
+        
```

Comment:
```suggestion

```

Reply:
Welcome to Airflow and thanks for your contribution!
It seems that you have an extra tab which causes the linters to fail.
Try removing it as suggest, and let's see that all pass smoothly :)